lib/workload/orcabus-stateless-stack.ts

```diff
@@ -41,10 +63,50 @@ export class OrcaBusStatelessStack extends cdk.Stack {
 
     // hook microservice construct components here
     this.createSequenceRunManager();
+
+    if (props.filemanagerDependencies) {
+      this.createFilemanager(props.filemanagerDependencies, this.vpc);
+    }
   }
 
   private createSequenceRunManager() {
     // TODO new SequenceRunManagerConstruct() from lib/workload/stateless/sequence_run_manager/deploy/component.ts
     //   However, the implementation is still incomplete...
   }
+
+  private createFilemanager(dependencies: FilemanagerDependencies, vpc: IVpc) {
+    // Opting to reconstruct the dependencies here, and pass them into the service as constructs.
+    const queue = Queue.fromQueueArn(
+      this,
+      'FilemanagerQueue',
+      Arn.format(
+        {
+          resource: dependencies.eventSourceQueueName,
+          service: 'Queue',
+        },
+        this
+      )
+    );
+    const databaseSecurityGroup = SecurityGroup.fromLookupByName(
+      this,
+      'FilemanagerDatabaseSecurityGroup',
+      dependencies.databaseSecurityGroupName,
+      vpc
+    );
```

Comment:
Similarly, you could reuse (which is commented around line 49-56) instead. You may then reuse along like `this.vpc` as class property.